spec.html

```diff
@@ -6497,7 +6498,7 @@ <h1>
           _F_: a constructor,
           optional _argumentsList_: unknown,
           optional _newTarget_: a constructor,
-        ): a Completion Record normally containing an ECMAScript language value
+        ): a Completion Record normally containing an Object
```

Comment:
Only if we also change `[[Construct]]` to return "a Completion Record normally containing an Object", right?

Reply:
Yes, sorry, those should change as well.